src/server/system_services/account_server.js

```diff
@@ -559,11 +562,7 @@ function reset_password(req) {
     }
 
     const params = req.rpc_params;
-    return verify_authorized_account(req)
-        .then(res => {
-            if (!res) throw new RpcError('UNAUTHORIZED', 'Invalid verification password');
-        })
-        .then(() => bcrypt_password(params.password.unwrap()))
+    return bcrypt_password(params.password.unwrap())
```

Comment:
since you are already touching this function, maybe break this to async\await as well